frontend/class-frontend.php

```diff
@@ -2011,42 +2015,106 @@ private function fetch_social_profiles() {
 	 *
 	 * @since 1.5.7
 	 *
-	 * @link  https://developers.google.com/webmasters/structured-data/slsb-overview
+	 * @link  https://developers.google.com/structured-data/site-name
 	 */
-	public function internal_search() {
+	public function website() {
+		$output = '{ "@context": "http://schema.org",
```

Comment:
If this is json, why not creating array with key => values and doing `json_encode`. That will be more readable.